### PR TITLE
W-15481980-update-mule-patches-rtf-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 1]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
+
+The following table describes the autoupgrade behavior of Mule runtime patch updates:
+
+[%header%autowidth.spread]
+|===
+|Mule runtime patch updates | Releases before October 3, 2023 | Releases after October 3, 2023
+| Versions 4.4 and previous |The deployment defaults to or autoupgrades to the latest patch of the selected minor. | Anypoint Platform API requires passing a valid and supported build tag in the deployment specification. If you pass a wrong build tag, the deployment fails.
+
+
+If you deploy a Mul app without the build tag, the deployment defaults to or autoupgrades to the latest patch of the selected minor.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
If you deploy a Mule app without the build tag, the deployment defaults to or autoupgrades to the latest patch of the selected minor.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 2]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
 During the grace period, you can select earlier date-patch versions until they are no longer available through the Runtime Manager UI but still usable through the API. Note that this behavior changes starting with Mule runtime engine version 4.5 with Edge and LTS channel releases. All build tags are available in the Runtime Manager UI and API.
 
+=== Autoupgrade Behavior
+
+The following table describes the autoupgrade behavior of Mule runtime patch updates:
+
+[%header%autowidth.spread]
+|===
+|Mule runtime patch updates | Releases before October 3, 2023 | Releases after October 3, 2023
+| Versions 4.4 and previous |The deployment defaults to or autoupgrades to the latest patch of the selected minor. | Anypoint Platform API requires passing a valid and supported build tag in the deployment specification. If you pass a wrong build tag, the deployment fails.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
| Versions 4.4 and earlier |The deployment defaults to or autoupgrades to the latest patch of the selected minor. | Anypoint Platform API requires passing a valid and supported build tag in the deployment specification. If you pass a wrong build tag, the deployment fails.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 3]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
+
+
+If you deploy a Mul app without the build tag, the deployment defaults to or autoupgrades to the latest patch of the selected minor.
+| Versions 4.5 and following | - | Mule runtime image names have the build tag built within them, so you do not have to pass a separate build tag. You are not defaulted to or autoupgraded to the latest patch of the selected minor.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
| Versions 4.5 and later | - | Mule runtime image names have the build tag built within them, so you do not have to pass a separate build tag. You are not defaulted to or autoupgraded to the latest patch of the selected minor.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 4]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
-
 During the grace period, you can select earlier date-patch versions until they are no longer available through the Runtime Manager UI but still usable through the API. Note that this behavior changes starting with Mule runtime engine version 4.5 with Edge and LTS channel releases. All build tags are available in the Runtime Manager UI and API.
 
+=== Autoupgrade Behavior
+
+The following table describes the autoupgrade behavior of Mule runtime patch updates:
+
+[%header%autowidth.spread]
+|===
+|Mule runtime patch updates | Releases before October 3, 2023 | Releases after October 3, 2023
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/runtime-patch-updates.adoc:40]: Behaviour

Comment (SparshAgarwal) [reply modules/ROOT/pages/runtime-patch-updates.adoc:40]: Behaviour before October 3, 2023

Comment (SparshAgarwal) [reply modules/ROOT/pages/runtime-patch-updates.adoc:40]: Behaviour after October 3, 2023

**Review by SparshAgarwal — APPROVED** [review]

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 5]
```diff
@@ -4,10 +4,6 @@
 The following provides important information about Mule runtime engine patch updates and the Mule runtime engine release channels for Anypoint Runtime Fabric.
 
 
-== Version Rewrite
```
Comment (jiahongli18) [inline modules/ROOT/pages/runtime-patch-updates.adoc:7]: Why are we removing this?

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 6]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
 During the grace period, you can select earlier date-patch versions until they are no longer available through the Runtime Manager UI but still usable through the API. Note that this behavior changes starting with Mule runtime engine version 4.5 with Edge and LTS channel releases. All build tags are available in the Runtime Manager UI and API.
 
+=== Autoupgrade Behavior
+
+The following table describes the autoupgrade behavior of Mule runtime patch updates:
+
+[%header%autowidth.spread]
+|===
+|Mule runtime patch updates | Releases before October 3, 2023 | Releases after October 3, 2023
+| Versions 4.4 and earlier |The deployment defaults to or autoupgrades to the latest patch of the selected minor. | Anypoint Platform API requires passing a valid and supported build tag in the deployment specification. If you pass a wrong build tag, the deployment fails.
```
Comment (jiahongli18) [inline modules/ROOT/pages/runtime-patch-updates.adoc:41]: This is not fully accurate. I would say what we had earlier was more accurate. More specifically (this is just an explanation, not a suggestion on what to add):

`If you pass a wrong build tag, the deployment fails.`
- This doesn't capture all the cases. If a build tag that was out of support was used or no build tag was passed, then the build tag used to deploy will be the first minimally supported one.
- The deployment does not fail (in <4.4 nor >=4.5) in the above case. However, if a _non-existent_ build tag is passed, then the deployment fails

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 7]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
+
+
+If you deploy a Mule app without the build tag, the deployment defaults to or autoupgrades to the latest patch of the selected minor.
```
Comment (jiahongli18) [inline modules/ROOT/pages/runtime-patch-updates.adoc:44]: Again, if you deploy an app for the first time without a build tag, I wouldn't say the right word to describe it is "autoupgrade"

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 8]
```diff
@@ -33,11 +29,25 @@ Runtime Fabric runs each application and gateway deployment on a separate Mule r
...
+| Versions 4.5 and later | - | Mule runtime image names have the build tag built within them, so you do not have to pass a separate build tag. You are not defaulted to or autoupgraded to the latest patch of the selected minor.
```
Comment (jiahongli18) [inline modules/ROOT/pages/runtime-patch-updates.adoc:45]: `Mule runtime image names have the build tag built within them, so you do not have to pass a separate build tag.`
- What is this trying to say?

`You are not defaulted to or autoupgraded to the latest patch of the selected minor.`
- Is this really accurate? if you don't provide a tag, the latest one is used.

**Pull request closed without merging**